EULA.md

```diff
@@ -0,0 +1,335 @@
+CONTENT.
+
+## Provider's Right to Update Services
+
+The Provider reserves the right at any time to modify this Agreement and
+to impose new or additional terms or conditions on your use of the
+Services. Any material modifications and additional terms and conditions
+will be communicated to you and, if accepted, will be effective
+immediately and will be incorporated into this Agreement. In the event
+that you refuse to accept such changes, the Provider will have the right
+to terminate this Agreement pursuant to Section 12.
+
+## Termination of the Agreement
+
+This Agreement may be terminated by you at any time by discontinuing your use
+of the Service and deleting your User Content or by written notice to the
+Provider's technical/support contact listed in the [contact information
+section](#contact-information). In such an event, the Provider may immediately
+terminate your account, but it may take up to 10 business days to do so. You
+should ensure prior to termination that you have transferred all data stored on
+the Services to another storage medium.
+
+This Agreement may be terminated by the Provider for any reason upon 48 hours’
+notice to you. In such an event, you will be afforded a reasonable opportunity
+to retrieve your User Content from the Services. If your account access has
+been suspended pursuant to Section 6, you may request to retrieve your User
+Content from the Services by emailing Provider's technical/support contact
+listed in the [contact information section](#contact-information). The Provider
+will take reasonable steps to arrange for you to retrieve such content,
+provided retrieval does not require an unreasonable amount of Provider's time
+or resources and that the content is not illegal; is not unlawfully
+pornographic or indecent; does not depict extreme acts of violence; does not
+advocate bigotry or hatred against any person or group of people based on their
+race, religion, ethnicity, sex, gender identity, sexual preference, disability,
+or impairment; and does not violate the privacy or infringe the rights of
+others). Any such retrieval shall be at your own expense.
+
+## Severability
+
+If any part of this Agreement is declared unenforceable or invalid, the
+remainder will continue to be valid and enforceable.
+
+## Governing Law
+
+This Agreement, and the interpretation, construction and enforceability
+hereof, and all rights and obligations of both you and the Provider,
+whether arising under this Agreement or otherwise, shall be governed by
+and construed in accordance with the laws of the Commonwealth of
+Massachusetts, without giving effect to its principles of conflict of
+laws. The sole and exclusive jurisdiction and venue for resolution of
+any disputes under this Agreement shall be in the state and federal
+courts located in Boston, Massachusetts, and you hereby submit to, and
+waive objection to, the jurisdiction and venue of such courts.
+
+## Contact Information
```

Comment:
This section was previously titled, "Information to be added in the link", which didn't make any sense. I've renamed it to "Contact Information", but I'm not wedded to that.